### PR TITLE
apps/testcase : Add binary manager utc

### PR DESCRIPTION
**Pull request by YiHyunJin**
Added binary manager utc.

TASH>>binary_manager_utc
########## Binary Manager UTC Start ##########

[utc_binary_manager_update_binary_p] PASS

[utc_binary_manager_update_binary_n] PASS

[utc_binary_manager_get_update_info_p] PASS

[utc_binary_manager_get_update_info_n] PASS

[utc_binary_manager_get_update_info_all_p] PASS

[utc_binary_manager_get_update_info_all_n] PASS

### CODE REVIEW

File: apps/examples/testcase/ta_tc/binary_manager/utc/Kconfig [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config EXAMPLES_TESTCASE_BINARY_MANAGER_UTC
+	bool "Binary Mnager UTC TestCase Example"
```
Comment (jeongchanKim) [inline apps/examples/testcase/ta_tc/binary_manager/utc/Kconfig]: Mnager -> Manager

**Comment by seinfra** [comment]
Target : [768a45ac241e985dfb066f1897f08e6ce8c1e349] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [768a45ac241e985dfb066f1897f08e6ce8c1e349] - Code Rule Check OK.

File: apps/examples/testcase/ta_tc/binary_manager/utc/utc_binary_manager_main.c [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <unistd.h>
```
Comment (jeongchanKim) [inline apps/examples/testcase/ta_tc/binary_manager/utc/utc_binary_manager_main.c]: Please check that headers are necessary or not.

File: apps/examples/testcase/ta_tc/binary_manager/utc/utc_binary_manager_main.c [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+	TC_SUCCESS_RESULT();
+}
+
+static void utc_binary_manager_get_update_info_p(void)
+{
+	int ret;
+	binary_update_info_t binary_info;
+
+	ret = binary_manager_get_update_info(APP_NAME, &binary_info);
+	TC_ASSERT_EQ("binary_manager_get_update_info", ret, OK);
```
Comment (jeongchanKim) [inline apps/examples/testcase/ta_tc/binary_manager/utc/utc_binary_manager_main.c]: Please check the real result, for example app name and so on.

**Comment by seinfra** [comment]
Target : [8f1d4141e22990b33a90e9a880efab4bc81d70c2] - Code Rule Check OK.

File: apps/examples/testcase/ta_tc/binary_manager/utc/utc_binary_manager_main.c [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+	}
+
+	utc_binary_manager_update_binary_n();
+	utc_binary_manager_update_binary_p();
+
+	utc_binary_manager_get_update_info_n();
+	utc_binary_manager_get_update_info_p();
+
+	utc_binary_manager_get_update_info_all_n();
+	utc_binary_manager_get_update_info_all_p();
```
Comment (sunghan-chang) [inline apps/examples/testcase/ta_tc/binary_manager/utc/utc_binary_manager_main.c]: You have to name it like "utc_binarymanager_xxxx". (Need to remove '_' at binary_manager.

**Pull request closed without merging**